### PR TITLE
doc(pendo): dbt source README edit for Fivetran style

### PR DESCRIPTION
**Pull request by fivetran-chloe**
Links https://fivetran.atlassian.net/browse/RD-129041 

Edits for Fivetran style. Rewords for clarity. Please address my question (inline) and check my edits for technical accuracy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,9 +37,9 @@ vars:
 
 ### Passthrough Columns
 
-Additionally, this package includes all source columns defined in the macros folder. We highly recommend including custom fields in this package as models now only bring in the standard fields for the `EVENT`, `FEATURE_EVENT`, `PAGE_EVENT`, `ACCOUNT_HISTORY`, and `VISITOR_HISTORY` tables.
+This package includes all of the source columns that are defined in the macros folder. We recommend including custom fields in this package because the staging models only bring in the standard fields for the `EVENT`, `FEATURE_EVENT`, `PAGE_EVENT`, `ACCOUNT_HISTORY`, and `VISITOR_HISTORY` tables.
```
[the remark this answers is not in the document]
Comment (fivetran-jamie) [reply README.md:40]: yah we do -- "column" might be the more straightforward choice

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ This package enriches your Fivetran data by doing the following:
 ## Models
 
 This package contains staging models, designed to work simultaneously with our [Pendo modeling package](https://github.com/fivetran/dbt_pendo). The staging models:
```
[the remark this answers is not in the document]
Comment (fivetran-jamie) [reply README.md:13]: the code just hasn't been merged into `main` yet -- once this PR https://github.com/fivetran/dbt_pendo/pull/1 is merged, the project this links to will be populated

File: README.md [hunk 3]
```diff
@@ -37,9 +37,9 @@ vars:
...
 
-You can add more columns using our passthrough column variables. These variables allow the passthrough fields to be aliased (`alias`) and casted (`transform_sql`) if desired, although it is not required. Datatype casting is configured via a SQL snippet within the `transform_sql` key. You may add the desired SQL snippet while omitting the `as field_name` part of the casting statement - this will be dealt with by the alias attribute - and your custom passthrough fields will be casted accordingly.
+You can add more columns using our passthrough column variables. These variables allow the passthrough fields to be aliased (`alias`) and casted (`transform_sql`) if you want, although it is not required. You can configure datatype casting by using a SQL snippet within the `transform_sql` key. You may add the desired SQL snippet while omitting the `as field_name` part of the casting statement - this will be dealt with by the alias attribute - and your custom passthrough fields will be casted accordingly.
```
[the remark this answers is not in the document]
Comment (fivetran-chloe) [reply README.md:42]: Thanks! Will reword to make that clear

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (fivetran-chloe) [reply README.md:40]: Thanks - I'll make the change

**Pull request merged**